C_CPP/Data Structures/Stack/Infix, Prefix & Postfix (Application of Stack)/infixEvaluation.cpp

```diff
@@ -0,0 +1,137 @@
+//Program to evaluate infix expression
+#include <bits/stdc++.h>
```

Comment:
Try adding more suitable comments to make it easily understandable.

Reply:
Ok sir

Reply:
Sir, I made all changes please review it again.